matter/wifi/rs911x/hal/efx_spi_intf.c

```diff
@@ -139,7 +139,19 @@ void sl_wfx_host_init_bus(void)
   GPIO->EUSARTROUTE[1].ROUTEEN = GPIO_EUSART_ROUTEEN_RXPEN |    // MISO
                                  GPIO_EUSART_ROUTEEN_TXPEN |    // MOSI
                                  GPIO_EUSART_ROUTEEN_SCLKPEN;
-
+  /*For MG24 DISPLAY,FLASH & RS9116 communication uses same SPI line which
+   * causes RS9116 communication interruption randomly. Below changes resolve 
+   * this issue. Once proper fix is given to multiplex all the features on SPI line, 
+   * these changes can be removed
+   */
+  //Enable RS916 communication
+  GPIO_PinModeSet(gpioPortC, 0, gpioModePushPull, 0);
+  //Disable flash
+  GPIO_PinModeSet(gpioPortC, 4, gpioModePushPull, 1);
+  //Disable Display
+  GPIO_PinModeSet(gpioPortC, 8, gpioModePushPull, 1);
+  //Disable Display Enable
+  GPIO_PinModeSet(gpioPortC, 9, gpioModePushPull, 0);
```

Comment:
Aren't those configuration overriden with `initUsart2()` in `rsi_hal_mcu_platform_init` ?. More over this is not the place to disable SPI pins for other devices. Please disable them properly in the CSA Matter repo by setting `enable_lcd = false` for the LCD and finding the right way to disable the external flash.

Reply:
the flags to use to disable the LCD and the external flash are :
`disable_lcd` , `show_qr_code` and `use_external_flash`